samples/packages/lake-formation-tag-sync/src/main/resources/package.pkl

```diff
@@ -0,0 +1,94 @@
+/* SPDX-License-Identifier: Apache-2.0
+   Copyright 2024 Atlan Pte. Ltd. */
+amends "modulepath:/Config.pkl"
+import "pkl:semver"
+import "modulepath:/BuildInfo.pkl"
+
+packageId = "@csa/lake-formation-tag-sync"
+packageName = "Lake Formation Tag Sync"
+version = semver.Version(BuildInfo.VERSION)
+description = "Add custom metadata properties to asset based upon the Lake Formation Tags."
+iconUrl = "http://assets.atlan.com/assets/ph-flow-arrow-light.svg"
+docsUrl = "https://solutions.atlan.com/lake-formation-tag-sync/"
+implementationLanguage = "Kotlin"
+containerImage = "ghcr.io/atlanhq/\(name):\(BuildInfo.VERSION)"
+containerImagePullPolicy = if (BuildInfo.VERSION.endsWith("-SNAPSHOT")) "Always" else "IfNotPresent"
+containerCommand {
+  "/dumb-init"
+  "--"
+  "java"
+  "com.atlan.pkg.lftag.LakeTagSynchronizer"
+}
+outputs {
+  files {
+    ["debug-logs"] = "/tmp/debug.log"
+  }
+}
+keywords {
+  "kotlin"
+  "utility"
+}
+preview = true
+
+uiConfig {
+  tasks {
+    ["Source"] {
+      description = "Inputs"
+      inputs {
+        ["import_type"] = new Radio {
+          title = "Import lake tags from"
+          helpText = "Select how you want to provide the file(s) containing the tags to be synched."
+          possibleValues {
+            ["DIRECT"] = "Direct file upload(s)"
+            ["CLOUD"] = "Object storage"
+          }
+          default = "DIRECT"
+        }
```

Comment:
Looks like from the code you don't want to allow a direct file upload option... I'm not sure you need this radio widget at all; but if you need a fixed value of `CLOUD` here for the `Utils.getInputFiles()` to work (?) you can create a hidden `TextInput` with the same name (`import_type`) and a default value of `CLOUD` — then there's no chance of any mistake on the users' part, and one less thing for them to click in the UX.

Reply:
Done